server/bell_oauth_profile.js

```diff
@@ -17,16 +17,16 @@ function profile (credentials, params, get, profileCb) {
     // Bell returns the parsed data and handles errors internally
     log.verbose('exchanged tokens for profile data:' + JSON.stringify(data));
     // TODO use Joi to validate `data` before sending to DB
-    db.getUserById(data.uid, function (err, user) {
+    user.get(data.uid, function (err, user) {
       if (err) {
-        log.warn('getUserById failed: ' + err);
-        return profileCb('getUserById failed: ' + err);
+        log.warn('user.get failed: ' + err);
+        return profileCb('user.get failed: ' + err);
```

Comment:
What's the signature for `profileCb()`? Should this just be returning a `profileCb(err)`?